data_generation/subgraphs_methods.py

```diff
@@ -101,4 +104,24 @@ def random_walk_neighbor_selection(full_net, starting_bus, subgraph_length):
 
     return subgraph_busses
 
+def number_changes(full_net,starting_bus,subgraph_length):
+    test_net = pp.pandapowerNet(full_net.copy())
+    # We vary every value based on how big they are around their point
+    # for i in range(len(test_net.bus)):
+        # test_net.bus.at[i,'vn_kv'] = np.random.normal(test_net.bus.at[i,'vn_kv'], np.sqrt(abs(test_net.bus.at[i,'vn_kv'])/100)) #goes from 0 - 350
+        # fullnet.bus.at[i,'max_vm_pu'] = np.random.normal(fullnet.bus.at[i,'max_vm_pu'], np.sqrt(abs(fullnet.bus.at[i,'max_vm_pu'])))
+        # fullnet.bus.at[i,'min_vm_pu'] = np.random.normal(fullnet.bus.at[i,'min_vm_pu'], np.sqrt(abs(fullnet.bus.at[i,'min_vm_pu'])))
+    for i in range(len(test_net.gen)):
+        test_net.gen.at[i,'p_mw'] = np.abs(np.random.normal(test_net.gen.at[i,'p_mw'], np.sqrt(abs(test_net.gen.at[i,'p_mw'])/100))) #goes from 0 - 1000
+        # test_net.gen.at[i,'vm_pu'] = np.abs(np.random.normal(1, 1/100)) #goes from 0 - 1.1
+        # fullnet.gen.at[i,'min_q_mvar'] = np.random.normal(fullnet.gen.at[i,'min_q_mvar'], np.sqrt(abs(fullnet.gen.at[i,'min_q_mvar'])))
+        # fullnet.gen.at[i,'max_q_mvar'] = np.random.normal(fullnet.gen.at[i,'max_q_mvar'], np.sqrt(abs(fullnet.gen.at[i,'max_q_mvar'])))
+        # fullnet.gen.at[i,'min_p_mw'] = np.random.normal(fullnet.gen.at[i,'min_p_mw'], np.sqrt(abs(fullnet.gen.at[i,'min_p_mw'])))
+        # fullnet.gen.at[i,'max_p_mw'] = np.random.normal(fullnet.gen.at[i,'max_p_mw'], np.sqrt(abs(fullnet.gen.at[i,'max_p_mw'])))
+    for i in range(len(test_net.load)):
+        test_net.load.at[i,'p_mw'] = np.abs(np.random.normal(test_net.load.at[i,'p_mw'], np.sqrt(abs(test_net.load.at[i,'p_mw']))))# goes from 0 - 500 (most under 100)
+        test_net.load.at[i,'q_mvar'] =np.abs(np.random.normal(test_net.load.at[i,'q_mvar'], np.sqrt(abs(test_net.load.at[i,'q_mvar'])))) # goes from -50 to about 300
+
+    return test_net
```

Comment:
I dont like having too many comments, I am ok if we will them (maybe add a comment saying why they will be needed and so we should keep them), if its leftover just remove them